src/silo/query_engine/query_engine.cpp

```diff
@@ -42,16 +42,16 @@ QueryResult QueryEngine::executeQuery(const std::string& query_string) const {
    int64_t filter_time;
    {
       const BlockTimer timer(filter_time);
-      const tbb::blocked_range<size_t> range(0, database.partitions.size(), 1);
-      tbb::parallel_for(range.begin(), range.end(), [&](const size_t& partition_index) {
+      for (size_t partition_index = 0; partition_index != database.partitions.size();
+           partition_index++) {
```

Comment:
Is this intentionally still here?

Reply:
Actually yes, while the exception were still causing me some headache